src/views/account/Homeworks/Document.tsx

```diff
@@ -164,9 +185,255 @@ const HomeworksDocument: Screen<"HomeworksDocument"> = ({ route }) => {
+            <MemoizedNativeItem>
+              <MemoizedNativeText variant="subtitle" numberOfLines={1}>
+                Nom de la matière
+              </MemoizedNativeText>
+              <View
+                style={{
+                  marginTop: 5,
+                  borderWidth: 1,
+                  borderColor: theme.colors.border,
+                  borderRadius: 8,
+                  overflow: "hidden",
+                }}
+              >
+                <Picker
+                  selectedValue={selectedPretty?.pretty}
+                  onValueChange={(itemValue) => {
+                    const selectedSubject = Object.entries(localSubjects).find(
+                      ([, subject]) => subject.pretty === itemValue
+                    );
+
+                    if (selectedSubject) {
+                      setSelectedPretty(selectedSubject[1]);
+                    }
+                  }}
+                  style={{
+                    color: theme.colors.text,
+                  }}
+                >
+                  {Object.entries(localSubjects).map(([key, subject]) => (
+                    <Picker.Item
+                      key={key}
+                      label={subject.pretty}
+                      value={subject.pretty}
+                    />
+                  ))}
+                </Picker>
+              </View>
+            </MemoizedNativeItem>
+          </MemoizedNativeList>
+        </View>
+
+        <MemoizedNativeList style={{ marginTop: 16 }}>
+          <MemoizedNativeItem>
+            <MemoizedNativeText variant="subtitle" numberOfLines={3}>
+              Contenu du devoir
+            </MemoizedNativeText>
+            <ResponsiveTextInput
+              style={{
+                fontFamily: "medium",
+                fontSize: 16,
+                color: theme.colors.text,
+              }}
+              value={contentHomework ?? ""}
+              onChangeText={(input) => {
+                if (input === "") {
+                  setContentHomework(null);
+                } else {
+                  setContentHomework(input);
+                }
+              }}
+            />
+          </MemoizedNativeItem>
+        </MemoizedNativeList>
+
+        {showDatePicker && (
+          <DateTimePicker
+            value={new Date(dateHomework)}
+            mode="date"
+            display="default"
+            onChange={(_event, selectedDate) => {
+              setShowDatePicker(false);
+              if (selectedDate) {
+                selectedDate.setHours(0, 0, 0, 0);
+                setDateHomework(selectedDate.getTime());
+              }
+            }}
+          />
+        )}
+
+        <ButtonCta
+          value="Valider"
+          onPress={() => {
+            if (!selectedPretty || !contentHomework) {
+              Alert.alert("Veuillez remplir tous les champs avant de valider.");
+              return;
+            }
```

Comment:
```suggestion
        <ButtonCta
          value="Valider"
          disabled={!selectedPretty || !contentHomework}
          onPress={() => {
```

Autant utiliser directement ce props

Reply:
merci, pask utiliser `Alert.alert` était très moche mdrr